### PR TITLE
Google maps feature and integrate other approved pull requests

### PR DESCRIPTION
**Pull request by joekinney-png**

### CODE REVIEW

File: client/App.jsx [hunk 1]
```diff
@@ -1,17 +1,18 @@
-import React, { Component } from 'react';
-import { Redirect, Route, Switch } from 'react-router-dom';
```
Comment (kklochan) [inline client/App.jsx:2]: I think we want to keep importing redirect, right?

Comment (joekinney-png) [reply client/App.jsx:2]: this was a duplication

Comment (joekinney-png) [reply client/App.jsx:2]: you are right nm

**Review by kklochan — APPROVED** [review]
We no longer need redirect imported, so I'm approving the changes.

**Pull request merged**